### PR TITLE
Readme sprint1 update

### PR DESCRIPTION
**Pull request by JasonRitz**

### CODE REVIEW

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/README.md [hunk 1]
```diff
@@ -13,9 +13,14 @@ Each class holds all the revelant data for that object.
 ### View Controllers Folder 
 Contains all of the view controllers for each scene in the main.storyboard. 
 The view controller handels all user interaction.
+
+#### loginViewControler
+The login view controller handels the user interactions made on the login view screen found in the main storyboard.
```
Comment (mmcbee1) [inline Mobile Test Runner for Jama/Mobile Test Runner for Jama/README.md]: Typo: handles

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/README.md [hunk 2]
```diff
@@ -13,9 +13,14 @@ Each class holds all the revelant data for that object.
 ### View Controllers Folder 
 Contains all of the view controllers for each scene in the main.storyboard. 
 The view controller handels all user interaction.
+
+#### loginViewControler
+The login view controller handels the user interactions made on the login view screen found in the main storyboard. 
+It sets up the username, password, and instance text boxes. Handels the keyboard for user input, checks for valid input, and then sends a basic auth request to the REST API. On login fail it presents the user with an error message, on login sucess it passes off to the projectsView screen.
```
Comment (mmcbee1) [inline Mobile Test Runner for Jama/Mobile Test Runner for Jama/README.md]: Typos: handles, success

File: Mobile Test Runner for Jama/Mobile Test Runner for Jama/README.md [hunk 3]
```diff
@@ -13,9 +13,14 @@ Each class holds all the revelant data for that object.
...
 The view controller handels all user interaction.
+
+#### loginViewControler
+The login view controller handels the user interactions made on the login view screen found in the main storyboard. 
+It sets up the username, password, and instance text boxes. Handels the keyboard for user input, checks for valid input, and then sends a basic auth request to the REST API. On login fail it presents the user with an error message, on login sucess it passes off to the projectsView screen. 
+
 ### Utils Folder
-* MaxTextLength.swift - Allows for setting of a max acceptible length in text boxes
-* RestHelper.swift - Is a class that makes all calls to the REST API using basic authorization. Calling the method hitEndPoint() will return the JSON data from the API request. All other methods are asynchronous helper methods to the hitEndPoint() method. 
+* MaxTextLength.swift - Allows for setting of a max acceptible length of input into text boxes. This field can be found in the atributes inspector panel for the text boxes in Xcode
```
Comment (mmcbee1) [inline Mobile Test Runner for Jama/Mobile Test Runner for Jama/README.md]: Typo: attributes

File: Mobile Test Runner for Jama/Mobile Test Runner for JamaTests/README.md [hunk 4]
```diff
@@ -1 +1,7 @@
-TEST README PLACEHOLDER
+# Unit Tests
+
+Mobile Test Runner for Jama was built using Test Driven Development. In the folders: ViewControllerUnitTests, utilsUnitTests, and ModelUnitTests you will find all of the unit test code used throughout development.
+
+
+## Running tests
+These tests can be run by hitting within Xcode, or by selecting Product/Test from the dropdown menu.
```
Comment (mmcbee1) [inline Mobile Test Runner for Jama/Mobile Test Runner for JamaTests/README.md]: Missing word? "by hitting" run/build/test? "within Xcode"

**Review by mmcbee1 — CHANGES_REQUESTED** [review]
Looks good, just a few small typos.

**Comment by JasonRitz** [comment]
fixed all of Meghan's requests

**Review by mmcbee1 — APPROVED** [review]

**Pull request merged**